src/temporal/common.ts

```diff
@@ -21,3 +24,37 @@ export const generateWorkflowId = (
 export interface InjectedProps {
   con: DataSource;
 }
+
+export enum TemporalError {
+  NotFound = 'WorkflowNotFoundError',
+}
+
+export const getWorkflowHandle = async (
+  workflowId: string,
+): Promise<WorkflowHandle> => {
```

Comment:
This can also be `WorkflowHandle | undefined` then, if `getHandle` can be `undefined`?

Reply:
Agreed. Let me update this one.